app/services/parsers/html.rb

```diff
@@ -1,7 +1,14 @@
 module Parsers
   class Html < Text
+    attr_reader :doc
+
+    def initialize(document)
+      super(document)
+      @doc = Nokogiri::HTML(@document.contents)
+    end
+
     def get_title
```

Comment:
Could this method be called just `title`?

Reply:
fixed****